pallets/liquidity-proxy/src/lib.rs

```diff
@@ -2197,11 +2193,8 @@ pub mod pallet {
     use frame_system::pallet_prelude::*;
 
     // TODO: #395 use AssetInfoProvider instead of assets pallet
-    // TODO: #441 use TradingPairSourceManager instead of trading-pair pallet
     #[pallet::config]
-    pub trait Config:
-        frame_system::Config + common::Config + assets::Config + trading_pair::Config
```

Comment:
You also should to remove `trading_pair` dependency from the `Cargo.toml` files. In all pallets.